core/workspace_svg.js

```diff
@@ -1838,7 +1839,8 @@ Blockly.WorkspaceSvg.prototype.showContextMenu = function(e) {
 
   // Allow the developer to add or modify menuOptions.
   if (this.configureContextMenu) {
-    this.configureContextMenu(menuOptions);
+    // pxt-blockly Also pass in the event for custom menuOptions
```

Comment:
We do this in core as well: https://github.com/google/blockly/blob/master/core/workspace_svg.js#L1810
Not pxt-blockly specific if you wanted to keep that updated.